tests/infrastructure/instance_types/test_cpu_memory_hotplug_instancetype.py

```diff
@@ -86,6 +90,18 @@ def hotplug_vm_snapshot_instance_type(instance_type_hotplug_vm):
         yield snapshot
 
 
+def hot_plug_cluster_instance_type(guest_sockets, cpu_model):
+    return VirtualMachineClusterInstancetype(
+        name=f"hot-plug-{guest_sockets}-cpu-instance-type",
+        memory={"guest": FOUR_GI_MEMORY},
+        cpu={
+            "guest": guest_sockets,
+            "model": cpu_model,
+            "maxSockets": EIGHT_CPU_SOCKETS,
+        },
+    )
```

Comment:
we set functions before fixtures in the file.

here, I would even move this function to utils file